README.md

```diff
@@ -1,2 +1,6 @@
-# python-dbuslib
-DBus library in Python 3
+# dasbus
+This DBus library is written in Python 3, based on GLib and inspired by pydbus.
+
+The code used to be part of the [Anaconda Installer](https://github.com/rhinstaller/anaconda)
+project. It was based on the [pydbus](https://github.com/LEW21/pydbus) library, but we replaced
+it with our own solution because of its inactivity. The dasbus library is a result of this effort.
```

Comment:
`inactivity` doesn't seems to me like a correct word here. Maybe `maintainer unresponsiveness`?